src/widgets/CategoryWidget.js

```diff
@@ -19,6 +19,7 @@ import { AggregationTypes } from './AggregationTypes';
   * @param  {formatterCallback} [props.formatter] - Function to format each value returned.
   * @param  {boolean} [props.viewportFilter=false] - Defines whether filter by the viewport or not. 
   * @param  {errorCallback} [props.onError] - Function to handle error messages from the widget.
+  * @param  {Object} [props.wrapperProps] - Extra props to pass to WrapperWidgetUI (see storybook for reference)
```

Comment:
Maybe this information is not clear enough. Please try to improve it putting a link or explaining that there you can put the props defined for the WrapperWidgetUI in this component documentation.

Reply:
Can you put a markdown link like [WrapperWidgetUI](https://storybook-react.carto.com/?path=/docs/widgets-wrapperwidgetui--default) in there?

Reply:
@neokore I added the link to the storybook in the JSDoc using markdown syntax